inferno/net.py

```diff
@@ -312,6 +312,7 @@ def __init__(
             batch_size=128,
             iterator_train=DataLoader,
             iterator_test=DataLoader,
+            train_split=CVSplit(0.2),
```

Comment:
I was not under the impression that a default split is our default case. Wouldn't you rather use grid searches primarily?